.github/workflows/master-to-app-engine.yml

```diff
@@ -69,7 +69,7 @@ jobs:
         run: python3 manage.py collectstatic --noinput --clear
 
       - name: Substitute environment variables
-        run: envsubst < .config/app.yaml.template > app.yaml
+        run: envsubst < .config/app.template.yaml > app.yaml
```

Comment:
Is this distinction important to make the deploy work (i.e. do you think the approach before wouldn't work)? We have a `.env.template` as opposed to `.template.env`, so for consistency I feel like the `.template` being at the end sounds better. Kinda pedantic though, so if there's a reason then this point is moot.

Reply:
You're not being pedantic at all! All comments are valuable.

I had not thought about `.env` files, but now that you've mentioned it, we can rename them to ensure proper extensions are used. (After all, this PR already changes a lot about environment variables anyways.) I would prefer using prefixes or infixes, since suffixes should be used for extensions (which, in turn, are used by IDEs for color coding and auto-formatting. Do you mind if I change as such?

Reply:
I'm not sure.... from my brief investigation, it seems as if `.env.template` seems to be the more common approach. In fact, if you Google exactly `".template.env"`, the [first page that comes up](https://dump-env.readthedocs.io/en/latest/) is about using `.env.template` files. However, adding to the confusion, it does mention `.template.env` files too, but those _appear_ to be used in a different context. I'm not sure that syntax highlighting is _terribly_ important for `.env` files considering they're just a series of `{NAME}={VALUE}` lines either.

Regardless on what we end up deciding, we should definitely strive for consistency.

Reply:
What I want is syntax highlighting for `app.yaml` because there is a huge difference:
![colorcodedyaml](https://user-images.githubusercontent.com/7676354/106096555-d027d680-6103-11eb-9d0a-7c6983147eef.png)
![plaintextyaml](https://user-images.githubusercontent.com/7676354/106096556-d027d680-6103-11eb-967c-b30020c2679f.png)

How about we make a compromise and use suffixes for plaintext files (which don't need formatting) and infixes for others? 🤔

Reply:
Reverted in 0c42f0367937333ecca53b3baf4561e23edc29c4 for now.